go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/go-ldap/ldif
+
+go 1.14
+
+require (
+	github.com/go-asn1-ber/asn1-ber v1.4.1 // indirect
+	github.com/go-ldap/ldap v3.0.3+incompatible
```

Comment:
Could this possibly be cleaned out - do we know why this line gets generated?

Reply:
Generated when running travis jobs/make files manually